conda-recipe/conda_build_config.yaml

```diff
@@ -6,7 +6,7 @@ cxx_compiler_version:       # [linux]
   - 7                       # [linux and x86_64]
 
 dpctl:
-  - 0.11.0
+  - 0.11.2
```

Comment:
Is `0.11.2` available?

Reply:
The policy is to build with first minor version (0.11.0) and test with any patch version (0.11.2). That is why 0.11.0.